examples/Evolution/NN3DWalkers.cpp

```diff
@@ -160,7 +160,8 @@ class NN3DWalkersExample : public NN3DWalkersTimeWarpBase
 	 m_filterCallback(NULL)
 	{
 		b3Clock clock;
-		srand(clock.getSystemTimeMilliseconds());					// Set random milliseconds based on system time
+		srand(clock.getTimeMilliseconds());					// Set random milliseconds based on system time
+		clock.reset(true);									// Reset clock to zero to get new random seed
```

Comment:
I thought I could basically reset it directly after using it, so that it is reset for the next call. But that is ok too.